pages/index/priority.vue

```diff
@@ -146,22 +141,34 @@ export default {
             'visualizationStage',
             'response',
             'params',
+            'tablePriority',
         ]),
     },
 
     methods: {
         search() {
-            if (this.visualizationStage == 'stage1') this.getFeatures()
-            if (this.visualizationStage == 'stage3') this.getDataTable()
+            if (this.tablePriority === true) this.getDataTable()
```

Comment:
`if else` is not following the code pattern;

Reply:
Improvement in the code and logic pattern, the old logic was comparing the total of polygons in the map with the total of the table if the amount of polygons in a period is equal, the map will not be updated generating inconsistency. New logic ;)